js/privatebin.js

```diff
@@ -43,10 +43,21 @@ jQuery.PrivateBin = function($, sjcl, Base64, RawDeflate) {
         var me = {};
 
         /**
+         * list of UserAgents (parts) known to belong to a bot
+         *
+         * @private
+         * @enum   {Object}
+         * @readonly
+         */
+        var BadBotUA = [
```

Comment:
Which bots do execute JS and are targeted by these pattern? (Asking so we can replicate this in the unit tests)